service/arm_templates/security_groups_setup.json

```diff
@@ -30,22 +21,23 @@
     "kubernetesAPISecurePort": {
       "type": "int"
     },
-    "etcdPort": {
-      "type": "int"
-    },
     "kubernetesIngressSecurePort": {
       "type": "int"
     },
     "kubernetesIngressInsecurePort": {
       "type": "int"
+    },
+    "azureHealthProbeCidr": {
+      "type": "string",
+      "defaultValue": "168.63.129.16/32"
```

Comment:
Azure uses this fixed public IP address from which it probes the load balancer backend pool members.

See this for more information: https://blogs.msdn.microsoft.com/mast/2015/05/18/what-is-the-ip-address-168-63-129-16/

Reply:
1) there is special tag "AzureLoadBalancer" that should be used for loadbalancer traffic. See link
https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-nsg

2) This is probably not needed at all, because LoadBalancer traffic opened by default.
See Inbound default rules in link above.

If it does not work for you, let's chat in slack, because i have this working in host cluster.